src/components/webcam.tsx

```
@@ -160,6 +166,7 @@ export const Webcam = () => {
           </div>
         </div>
       </div>
+      <MessageInputContainer initialTranscript={currentTranscript} />
```

ESLint

[error] 'MessageInputContainer' is not defined.

(react/jsx-no-undef)